itkzlib/zlib.rc

```diff
@@ -16,13 +16,13 @@ BEGIN
     //language ID = U.S. English, char set = Windows, Multilingual
 
     BEGIN
-      VALUE "FileDescription", "zlib data compression library\0"
-      VALUE "FileVersion",	"1.2.3\0"
+      VALUE "FileDescription", "zlib data compression and ZIP file I/O library\0"
+      VALUE "FileVersion",	"1.3.1\0"
       VALUE "InternalName",	"zlib\0"
-      VALUE "OriginalFilename",	"zlib.dll\0"
+      VALUE "OriginalFilename",	"zlibwapi.dll\0"
```

Comment:
This might not need to change. Might not be important, though.

Reply:
Just tell me which you want :) I know little of Windows. That's from https://github.com/madler/zlib/blob/develop/contrib/vstudio/vc17/zlib.rc